python/upt-cpan/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                upt-cpan
```

Comment:
This might need some discussion, but this installs just a set of library files, and no executable, right? (I need to check.)

If that is the case, I would be somewhat tempted to install this as `py-upt-cpan`. If you don't do that, you should move the `Portfile` under some different directory, I believe. Ports inside `python/` are kind of reserved for all ports with `py[version]-name` subports.

Reply:
I agree with the name change

Reply:
Okey, I will do the required name change for cpan, pypi and macports.

Should I change the location of upt port as the name of it cannot be changed?

Reply:
why can you not change the name of the port - what's wrong with ```py-utp```?

Reply:
If I change the name to py-upt then typing upt in cli won't be able to access it.
Even if we access it using upt-3.7, even then errors are there as upt imports upt.* in many places.

ref: https://lists.macports.org/pipermail/macports-dev/2019-April/040505.html

Reply:
I read that message, but in a later response you said that typing ```upt-3.7``` worked and that's the way it's supposed to be (at least in my understanding). There shouldn't be an issue with the imports as the library itself will still be called ```upt``` it's just the script in "bin" that changes the name. I can take a look later today what the exact problems are with it and will respond here.

Reply:
Sorry, my bad I retried installing using py-upt port and it works.